tsc/data/translations/fr.po

```diff
@@ -1,1605 +1,1728 @@
-# This file is distributed under the same license as the Secret Maryo Chronicles package.
-#
-# Copyright © 2008 Tester <joelekiffeur@yahoo.fr>
-# Copyright © 2008 Dylan Aïssi <bob.dybian@gmail.com>
-#
-# Translation of Secret Maryo Chronicles to French
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR The TSC Contributors
+# This file is distributed under the same license as the PACKAGE package.
+# 
+# Translators:
+# Benjamin Leduc <giby_the_kid@yahoo.fr>, 2015
 msgid ""
 msgstr ""
-"Project-Id-Version: Secret Maryo Chronicles\n"
+"Project-Id-Version: The Secret Chronicles of Dr. M., TSC\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2014-08-30 18:31+0200\n"
-"PO-Revision-Date: 2014-08-30 19:42+0200\n"
-"Last-Translator: Arno <petit.loup.2911@gmail.com>\n"
-"Language-Team: UNMAINTAINED\n"
```

Comment:
This should say something like "French TSC Translation team", and if there’s a mailinglist for them or a web URL, it should be noted here in angle brackets `<>`.